src/components/User/index.ts

```diff
@@ -1,2 +1,3 @@
 export type {UserSize, UserProps} from './types';
+export {DEFAULT_USER_SIZE} from './constants';
```

Comment:
Why we need it as a public constant?

Reply:
It can be useful for making some wrappers around the User component. I did this by analogy with the Avatar component.